src/client/components/priceAndTodaysPriceChange/index.test.js

```diff
@@ -0,0 +1,36 @@
+import { expect } from 'chai';
+
+import { mapStateToProps } from '.';
+import * as calculatePriceChange
+  from '../../../utils/stockDataUtils/calculatePriceChange';
+import * as selectors from '../../selectors';
+import * as numberFormatting
+  from '../../../utils/formatting/numberFormatting';
+
+describe('mapStateToProps', () => {
+  it('properly maps state to props', () => {
+    const mockCurrentPrice = 113.14125321;
+    const mockPriceChange = 1.26521123;
+
+    const mockState = {};
```

Comment:
can you put `mockCurrentPrice` and `mockPriceChange` into the mock state to make these tests more descriptive of what actually happens?